src/lib/components/Sidebar.svelte

```
@@ -0,0 +1,177 @@
+                    <line x1="8" y1="2" x2="8" y2="6"/>
+                    <line x1="3" y1="10" x2="21" y2="10"/>
+                </svg>
+                <span class="sidebar-btn-label">カレンダー</span>
+            </a>
+        {/if}
+    </nav>
+
+    <!-- 下部: テーマ・プロフィール・設定・ログアウト -->
+    <div class="sidebar-bottom">
+        <!-- テーマ切替 -->
+        <button class="sidebar-btn" onclick={toggleTheme} aria-label="テーマ切替" title={theme === 'dark' ? 'ライトモードに切替' : 'ダークモードに切替'}>
```

Biome (2.4.13)

[error] Provide an explicit type attribute for the button element.

(lint/a11y/useButtonType)